File: ce/customer-engagement/on-premises/snippets/visualbasic/CRMV8/queryvb/vb/createalinqquery2.vb
' <snippetcreatealinqquery2>


 Dim queryAccounts = From a In svcContext.AccountSet _
                                    Join owner In svcContext.SystemUserSet _
                                    On a.OwnerId.Id Equals owner.SystemUserId _
                                    Where owner.LastName IsNot "Cannon" _
                                    Select New Account With _
                                           {
                                               .Name = a.Name, _
                                               .Address1_City = a.Address1_City _
                                           }
 Console.WriteLine("Accounts not owned by user w/ last name 'Cannon'")
 Console.WriteLine("================================================")
 For Each a In queryAccounts
  Console.WriteLine(a.Name &amp; " " &amp; a.Address1_County)
 Next a

' </snippetcreatealinqquery2>

File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/query/cs/createalinqquery2.cs
// <snippetcreatealinqquery2>


 var queryAccounts = from a in svcContext.AccountSet
                     join owner in svcContext.SystemUserSet
                       on a.OwnerId.Id equals owner.SystemUserId
                     where owner.LastName != "Cannon"
                     select new Account
                     {
                      Name = a.Name,
                      Address1_City = a.Address1_City
                     };
 System.Console.WriteLine("Accounts not owned by user w/ last name 'Cannon'");
 System.Console.WriteLine("================================================");
 foreach (var a in queryAccounts)
 {
  System.Console.WriteLine(a.Name + " " + a.Address1_County);
 }

// </snippetcreatealinqquery2>